### PR TITLE
Client server login

### PR DESCRIPTION
**Pull request by BichraiX**
Added GET /login endpoint and fixed register from previous merge (#109 )

### CODE REVIEW

File: packages/matrix-client-server/src/login/getLogin.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { type expressAppHandler, send } from '@twake/utils'
+import type MatrixClientServer from '..'
+
+const getLogin = (clientServer: MatrixClientServer): expressAppHandler => {
+  return (req, res) => {
+    send(res, 200, clientServer.conf.login_flows)
```
Comment (guimard) [inline packages/matrix-client-server/src/login/getLogin.ts:6]: There is more than one authorized login flow. The more important for us is the "Openid-Connect". There are libraries to do this. I can explain the flow
@BichraiX @h1ppox99 @Mathixx

**Review by guimard — COMMENTED** [review]
What is the goal of this MR ? Only get login flow ?

**Comment by BichraiX** [comment]
> What is the goal of this MR ? Only get login flow ?

Mostly to correct the /register endpoint that was messed up in a previous merge request as well as to add the GET /login endpoints which returns all the login flows supported by the server

File: packages/matrix-client-server/src/login/getLogin.ts (hunk 1 above)
Comment (BichraiX) [reply packages/matrix-client-server/src/login/getLogin.ts:6]: Yes login_flows is a list of all authorized login flows. I've filled it with an example from the spec but will add Openid-Connect to it when I've understood what it is

**Pull request merged**